osu.Game/Overlays/VolumeOverlay.cs

```diff
@@ -97,6 +102,12 @@ protected override void LoadComplete()
                 volumeMeter.Bindable.ValueChanged += _ => Show();
 
             muteButton.Current.ValueChanged += _ => Show();
+
+            OverlayActivationMode.BindValueChanged(val =>
+            {
+                if (val.NewValue == OverlayActivation.Disabled)
+                    Hide();
```

Comment:
This is going to perform a one-time hide when the overlay activation mode changes to `Disabled`, but I'm pretty sure it will still permit the volume overlay to show when the mode is `Disabled`, e.g. when you mute/unmute/change volume? Was that the intention here?

Reply:
I wanted to hide it when the `OverlayActivation` changes, I thought that `OverlayActivation` should only control the ability to activate a overlay, not hiding a currently enabled one.
I'm confused on how to proceed: Should I just block any keyboard input when `OverlayActivation` is set to `Disabled` and let the overlay hide by itself after the delay or keep this code and let `OverlayActivation` control if the overlay can be opened again?